Homework_9/style.css

```diff
@@ -0,0 +1,83 @@
+img {
+  padding: 10px;
+  border: 2px solid red;
+  width: 100px;
+  height: 100px;
+  float: right;
+  
+}
+
+ #wrapper {
+ margin-bottom: 100px;
+} 
+
+
+
+p,
+h1,
+h3 {
+  
+  color: #333;
+  font-family: Arial, Helvetica, sans-serif;
+  border-bottom: 1px solid red;
+}
+
+h3 {
+  font-weight: bold;
+  margin-top: 0px;
+}
+
+h1 {
+  font-style: oblique;
+  
+}
+
+th {
+  text-transform: uppercase;
+}
+
+a:link {
+  color: #00f;
+  text-decoration: underline;
+}
+
+a:visited {
+  color: #005;
+  text-decoration: underline;
+}
+
+a:hover {
+  text-decoration: none;
+}
+
+ul {
+  list-style-type: square;
+}
+
+th {
+  background-color: #0066cc;
+}
+
+td {
+  background-color: #66b3ff;
+}
+
+table {
+  width: 100%;
+}
+
+.p-float {
+  float: right;
+  clear: rigth;
+}
+
+.ul-float {
```

Comment:
if you have a class name with `ul-FLOAT` don't assign other properties to such class

Reply:
or change the name of class